static/js/metrics-explorer.js

```diff
@@ -575,13 +578,15 @@ async function addQueryElement() {
             await addAlertsFormulaElement();
         }
     }
-
     // Show or hide the query close icon based on the number of queries
     updateCloseIconVisibility();
-
     setupQueryElementEventListeners(queryElement);
-
     queryIndex++;
+
+    // Apply stored settings to the new query element
+    await handleQueryAndVisualize(nextQueryName, queries[nextQueryName]);
```

Comment:
1. You don't need to call `handleQueryAndVisualize` to update the properties.
Also, it gives an error.
<img width="1271" alt="image" src="https://github.com/user-attachments/assets/b77c626f-6293-473e-a3a8-4e77510c2fd6">